public/data/job-openings-data.json

```diff
@@ -448,7 +448,7 @@
       "companyName": "DhiWise Pvt. Ltd.",
       "location": "Surat, India",
       "bio": "DhiWise is a programming platform where developers can convert their designs into developer-friendly code for mobile and web apps. DhiWise automates the application development lifecycle and instantly generates readable, modular and reusable code. At DhiWise, you can fall in love with your work again. A platform built for developers to build apps in a way that uses their engineering muscle rather than forcing the use of crammed information.",
-      "companyLogoUrl": "https://drive.google.com/file/d/1CZ2lSzbhLWTIeG6fsZ5UQYl8_qzCshdK/view",
+      "companyLogoUrl": "https://uploads-ssl.webflow.com/618e36726d3c0f19c9284e56/6392ef0773801c4452bd8bde_DhiWise-Logo-150X150.jpgw",
```

Comment:
Access denied, Please check at your end and then submit in PR.
Thanks

Reply:
<img width="941" alt="Screenshot 2022-12-09 at 4 32 35 PM" src="https://user-images.githubusercontent.com/16114199/206688278-14a114e7-6638-4d94-b139-1d1104aa0147.png">

Reply:
@JaldeepAsodariya I think the issue is typo in URL, "W" got appended at last https://uploads-ssl.webflow.com/618e36726d3c0f19c9284e56/6392ef0773801c4452bd8bde_DhiWise-Logo-150X150.jpg

Reply:
[typo error removed]